code/modules/mob/living/carbon/human/life.dm

```diff
@@ -610,6 +610,19 @@
 			if(prob(nausea))
 				delayed_vomit()
 
+		if(CE_ITCH in chem_effects)
+			var/itching = chem_effects[CE_NOITCH]
```

Comment:
Why are you using `NOITCH` first if you're trying to check for itching?

Reply:
Whoops. Mixed up `NOITCH` and `ITCH`. Should be good now.